modules/ROOT/nav.adoc

```diff
@@ -3,6 +3,7 @@
 * xref:design-modify-raml-specs-conform.adoc[Modify Published RAML API Specifications to Conform Completely to RAML 0.8 or 1.0]
  ** xref:design-common-problems-raml-08.adoc[Common Problems in Conforming to RAML 0.8]
  ** xref:design-common-problems-raml-10.adoc[Common Problems in Conforming to RAML 1.0]
+  *** xref:design-named-examples.adoc[Guide to Named Examples]
  ** xref:design-json-schema-required-error.adoc[Common Errors in Specifying Required Fields in JSON Schemas]
```

Comment:
@kbostrom , they are error messages and a few warning messages. I think "issues" has been drained of its meaning. People use it for many situations in which they prefer to use a euphemism rather than a more accurate word. The most egregious example is perhaps "Working at home today due to stomach issues."